packages/integration-tests/api/api.js

```diff
@@ -309,8 +309,35 @@ async function getExecutionStatus({ prefix, arn }) {
   return JSON.parse(payload.body);
 }
 
+async function postRule({ prefix, rule }) {
```

Comment:
Small issue but I don't think this should be called postRule, because it's a GET not a POST. Maybe getRule would be better?

Reply:
And is this list rules or getting a specific rule ?

Reply:
Is this old stuff? I'm just seeing the things you have in api/rules.js

Reply:
Wow, nice catch. I had been messing around with some things and forgot to back that out. I removed this and all of the actual code is in rules.js.